File: POOJAVA/src/clasesPOO/DocenteCurso.java
package clasesPOO;

public class DocenteCurso 
{
    String idDoc;
    String idPers;
    String idCurso;

    public DocenteCurso(String idD,String idCu)
    {
        establecerDatos(idD,idCu);
    }
    public String ObtenerDocente()
    {
       return idDoc;
    }
    public String ObtenerCurso()
    {
       return idCurso;
    }
    void  establecerDatos(String idDo,String idCur)
    {
        idDoc=idDo;
        idCurso=idCur;
    }
}

File: POOPHP/app/Model/DocenteCurso.php
<?php
namespace App\Model;
class DocenteCurso
{
    public $idDoc="";
    public $idPers="";
    public $idCurso="";

   public function __construct(String $idD,String $idCu)
    {
        $this->establecerDatos($idD,$idCu);
    }
    public function ObtenerDocente()
    {
        return $this->idDoc;
    }
    public function ObtenerCurso()
    {
        return $this->idCurso;
    }
    public function establecerDatos(String $idDo,String $idCur)
    {
        $this->idDoc=$idDo;
        $this->idCurso=$idCur;
    }
}
?>